### ISSUE TITLE
Profiler Search route is not correctly detected in NonChannelLocaleListener

### ISSUE THREAD

**Issue by thi3rry** [Sylius/Sylius#11362]
When you want to access the Profiler Last 10 requests or search for a profile profiler it throws 
an Exception The "en_US" locale is unavailable in this channel when channel doesn't have en_US as locale enable.

The profiler search&results routes are not correctly detected in :  https://github.com/Sylius/Sylius/blob/878ad6150455dbff2df0875e883ae94ce1e042bd/src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php#L67

`_profiler_search` and `_profiler_search_results` route should be ignore as `_profile`
```patch
Index: src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php
===================================================================
--- src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php
+++ src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php
@@ -57,7 +57,7 @@
         }
 
         $request = $event->getRequest();
-        if (in_array($request->attributes->get('_route'), ['_wdt', '_profiler'])) {
+        if (in_array($request->attributes->get('_route'), ['_wdt', '_profiler', '_profiler_search', '_profiler_search_results'])) {
             return;
         }
```

**Comment by pamil** [comment]
Fixed in #11364.

**Comment by thi3rry** [comment]
Thanks @pamil, I also created a PR on 1.7 branch since there was conflict, the merge on master should be automatically done.
